packages/react-menu/src/components/MenuList/MenuList.types.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utils';
+import { RefObjectFunction } from '@fluentui/react-hooks';
+
+export interface MenuListProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {}
+
+export interface MenuListState extends MenuListProps {
+  /**
+   * Ref to the root slot
+   */
+  ref: RefObjectFunction<HTMLElement>;
```

Comment:
```suggestion
  ref: React.MutableRefObject<HTMLElement>;
```

I would rather use a React as it will be easier to work with it (definitions are compatible)

Reply:
That's fair